### PR TITLE
Document scopes without selectors

### PR DESCRIPTION
**Pull request by dfabulich**
Incorporating changes from https://github.com/w3c/csswg-drafts/issues/7349

# REMEMBER: Attach this PR to the Trello card

### CODE REVIEW

**Review by mirisuzanne — APPROVED** [review]
This is great, thank you!

Not urgent, but we could also add a note to the `changes` section of `explainer.11tydata.yaml` (with credit, if you want it).

File: src/scope/explainer.11tydata.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@ created: 2020-12-15
 changes:
   - time: 2021-08-24
     log: Link to syntax comparison
+  - time: 2022-10-03
+    log: Document scopes without selectors
```
Comment (dfabulich) [inline src/scope/explainer.11tydata.yaml:8]: @mirisuzanne It's not clear to me how to add a credit link, e.g. to my Twitter or Github profile, or whether that's even appropriate here. Would you prefer just a parenthetical "(Thanks to Dan Fabulich)" here?

Comment (mirisuzanne) [reply src/scope/explainer.11tydata.yaml:8]: That's fine. Markdown syntax is also ok to use in the log. (and multiline logs are ok, Just start them the `|` and a line break: 

```yaml
changes:
  - time: 2021-08-24
    log: Link to syntax comparison
  - time: 2022-10-03
    log: |
      Mulitline log
      with [markdown](#etc)
```

**Comment by dfabulich** [comment]
Done!

**Review by mirisuzanne — APPROVED** [review]

**Comment by mirisuzanne** [comment]
Thank you so much!

**Pull request merged**